blender/arm/material/make_mesh.py

```diff
@@ -533,14 +533,14 @@ def make_forward(con_mesh):
         mrt = rpdat.rp_ssr  # mrt: multiple render targets
         if mrt:
             # Store light gbuffer for post-processing
-            frag.add_out('vec4 fragColor[2]')
+            frag.add_out('vec4 fragColor[3]')
             frag.add_include('std/gbuffer.glsl')
             frag.write('n /= (abs(n.x) + abs(n.y) + abs(n.z));')
             frag.write('n.xy = n.z >= 0.0 ? n.xy : octahedronWrap(n.xy);')
             frag.write('fragColor[0] = vec4(direct + indirect, packFloat2(occlusion, specular));')
             frag.write('fragColor[1] = vec4(n.xy, roughness, metallic);')
         else:
-            frag.add_out('vec4 fragColor[1]')
+            frag.add_out('vec4 fragColor[3]')
```

Comment:
Doesn't this also affect all non-refractive materials?

Reply:
I will have to deal with this in the same fashion than you did with the emissive